### PR TITLE
enh: URL checking for import web addresses

### LINKED ISSUE #2376: URL validator absent in the Get Data from URL input
**Describe the bug**

URL validator absent in the Get Data from URL input

**To Reproduce**
1. Go to 'Index page'
2. Click on 'import from web url'
3. Enter any random string eg:dnweferfe
4. Click on next and see the server tries to fetch the data from the wrong address

**Expected behavior**

There should be a URL Field validator which may restrain the user from entering random strings.
**Screenshots**

Screenshot 2020-03-09 at 5 14 06 AM

**Additional context**

cc: @thadguidry @wetneb If this is a valid enhancement .

### PR DESCRIPTION
**Pull request by kushthedude**
Fixes #2376

### CODE REVIEW

**Comment by kushthedude** [comment]
@wetneb Is it good to be merged ?

File: main/webapp/modules/core/scripts/index/default-importing-sources/sources.js [hunk 1]
```diff
@@ -83,7 +89,8 @@ UrlImportingSourceUI.prototype.attachUI = function(bodyDiv) {
   this._elmts.nextButton.html($.i18n('core-buttons/next'));
   
   this._elmts.nextButton.click(function(evt) {
-    if ($.trim(self._elmts.urlInput[0].value).length === 0) {
+    console.log(self._elmts.urlInput[0].value);
```
Comment (ostephens) [inline main/webapp/modules/core/scripts/index/default-importing-sources/sources.js]: We'd normally not leave console logging in code for release

File: main/webapp/modules/core/scripts/index/default-importing-sources/sources.js [hunk 2]
```diff
@@ -34,6 +34,12 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 function ThisComputerImportingSourceUI(controller) {
   this._controller = controller;
 }
+
+// Function to check if the URL getting entered is valid or not 
+function isUrlValid(url) {
+  // regex for a valid URL pattern
```
Comment (ostephens) [inline main/webapp/modules/core/scripts/index/default-importing-sources/sources.js]: Is there a source for this? If so, I think worth commenting where this came from so we can track back if any issues

**Comment by kushthedude** [comment]
The jquery url validator package. I will make comment its link

**Comment by kushthedude** [comment]
@wetneb @ostephens Done the changes. Good to be merged now!

**Pull request merged**